src/status_im/contexts/wallet/add_account/create_account/key_pair_name/view.cljs

```diff
@@ -0,0 +1,93 @@
+        [key-pair-name set-key-pair-name] (rn/use-state "")
+        [error set-error]                 (rn/use-state nil)
+        on-change-text                    (rn/use-callback
+                                           (fn [value]
+                                             (set-key-pair-name value)
+                                             (cond
+                                               (> (count value) key-pair-name-max-length)
+                                               (set-error :too-long)
+
+                                               (< 0 (count value) key-pair-name-min-length)
+                                               (set-error :too-short)
+
+                                               (validators/has-emojis? value)
+                                               (set-error :emoji)
+
+                                               (validators/has-special-characters? value)
+                                               (set-error :special-char)
+
+                                               :else (set-error nil))))
+        on-continue                       (rn/use-callback
+                                           (fn [_]
+                                             (case workflow
+                                               ;; TODO issue #19759. Implement creation account from
+                                               ;; private key
+                                               :import-private-key
+                                               (js/alert "Implement it!")
+
+                                               :new-key-pair
+                                               (rf/dispatch [:wallet/new-keypair-continue
+                                                             {:keypair-name key-pair-name}])
+                                               (js/alert "Unknown workflow")))
+                                           [workflow key-pair-name])
+        disabled?                         (or (some? error) (string/blank? key-pair-name))]
+    [rn/view {:style {:flex 1}}
+     [floating-button-page/view
+      {:customization-color customization-color
+       :header              [quo/page-nav
+                             {:icon-name           :i/arrow-left
+                              :on-press            navigate-back
+                              :accessibility-label :top-bar}]
+       :footer              [quo/button
+                             {:customization-color customization-color
+                              :disabled?           disabled?
+                              :on-press            on-continue}
+                             (i18n/label :t/continue)]}
+      [quo/page-top
+       {:title            (i18n/label :t/keypair-name)
+        :description-text (i18n/label :t/keypair-name-description)
+        :description      :text
+        :container-style  style/top}]
```

Comment:
Small comment 💅 

Maybe the name for this style definition could be a little more specific?
For example, something like `style/page-top`. Thoughts?

Reply:
Agree!